validator/client/src/main/java/tech/pegasys/teku/validator/client/doppelganger/DoppelgangerDetectionAction.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright ConsenSys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.client.doppelganger;
+
+import java.util.Set;
+import tech.pegasys.teku.bls.BLSPublicKey;
+
+public interface DoppelgangerDetectionAction {
+
+  default void shutDown() {
+    System.exit(1);
+  }
```

Comment:
Probably shouldn't be a default method given it's only used by one of two implementations. I'd just move it to the implementation.

Reply:
Done. I move the shutdown to the correspondent implementation